### ISSUE TITLE
pip3 not finding this package

### ISSUE THREAD

**Issue by jimkohl** [marklogic/python_api#96]
pip3 install marklogic_python_api

does not work (still).  read the 2017 comment about this but hey thought maybe by now thought it might work ...

anyways getting 
 ERROR: Could not find a version that satisfies the requirement marklogic_python_api (from versions: none)
ERROR: No matching distribution found for marklogic_python_api

I tried using the alternative suggested in the closed issue and got:

 pip3 install marklogic_python_api -i https://testpypi.python.org/pypi                                (master✱) 
Looking in indexes: https://testpypi.python.org/pypi
Collecting marklogic_python_api
  Downloading https://test-files.pythonhosted.org/packages/9e/85/bfb6b01dd1b68e9e44fefed3e5a67f22f96bb7fac42db7d4e161196bd427/marklogic_python_api-0.0.20.tar.gz (106kB)
     |████████████████████████████████| 112kB 303kB/s 
Requirement already satisfied: requests>=2.21.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from marklogic_python_api) (2.21.0)
Collecting requests_toolbelt>=0.9.1 (from marklogic_python_api)
  ERROR: Could not find a version that satisfies the requirement requests_toolbelt>=0.9.1 (from marklogic_python_api) (from versions: 0.4.0)
ERROR: No matching distribution found for requests_toolbelt>=0.9.1 (from marklogic_python_api)

**Comment by jimkohl** [comment]
I've resolved this however it would be much better to have this out in:  https://pypi.org/

**Comment by jrathgeber** [comment]
Yes please. This sounds like a good idea to me. Please accept my up vote.
